README.md

```diff
@@ -52,326 +53,8 @@ exciter.delete(primaryKey, tableName)
-Builds a DynamoDB update expression
-
-**Kind**: static method of <code>[Exciter](#Exciter)</code>  
-**Returns**: <code>String</code> - A property escaped expression for udpating DynamoDB.  
-
-| Param | Type | Description |
-| --- | --- | --- |
-| attributes | <code>Array</code> | An array of data attributes to update on an entity in DynamoDB. |
-
-<a name="Exciter.buildConditionExpression"></a>
-
-### Exciter.buildConditionExpression(conditions, groupOperator) ⇒ <code>String</code>
-Builds a DynamoDB conditional expression.
-
-**Kind**: static method of <code>[Exciter](#Exciter)</code>  
-**Returns**: <code>String</code> - A property escaped expression for querying DynamoDB.  
-
-| Param | Type | Description |
-| --- | --- | --- |
-| conditions | <code>Array</code> | A set of conditions from which to create a conditional expression. |
-| groupOperator | <code>String</code> | The logical operator which will join the conditions. Defaults to "AND". |
-
-<a name="Exciter.buildExpressionPlaceholders"></a>
-
-### Exciter.buildExpressionPlaceholders(attributes, substituionChar) ⇒ <code>Object</code>
-Builds expression name and value placeholders from a set of attributes. See
-http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ExpressionPlaceholders.html
-
-**Kind**: static method of <code>[Exciter](#Exciter)</code>  
-**Returns**: <code>Object</code> - A DynamoDB expression attribute values/names object.  
-
-| Param | Type | Description |
-| --- | --- | --- |
-| attributes | <code>Array</code> | An array of normalized expression attributes as returned by   normalizeExpressionAttribute(). |
-| substituionChar | <code>String</code> | What string to prepend to keys of the object. ex: `#` or `:` |
-
-<a name="Exciter.valueIsEmpty"></a>
-
-### Exciter.valueIsEmpty(value) ⇒ <code>boolean</code>
-Determines whether a value is empty according to DynamoDB.
-
-DynamoDB does not allow us to store null, undefined, empty string, or empty
-array values. This function can be paired with a filter to skip empty
-values when writing to DynamoDB.
-
-**Kind**: static method of <code>[Exciter](#Exciter)</code>  
-**Returns**: <code>boolean</code> - Whether or not the value is valid for DynamoDB storage.  
-
-| Param | Type | Description |
-| --- | --- | --- |
-| value | <code>mixed</code> | The value to validate. |
-
-
 ## Contributors
 
-[![Luke](https://avatars.githubusercontent.com/u/1127238?s=130)](https://github.com/infiniteluke) | [![Peter Sieg](https://avatars.githubusercontent.com/u/3128659?s=130)](https://github.com/chasingmaxwell)
---- | ---
-[Luke](https://github.com/infiniteluke) | [Peter Sieg](https://github.com/chasingmaxwell)
+[![](https://avatars.githubusercontent.com/u/?s=130)]() | [![Luke](https://avatars.githubusercontent.com/u/1127238?s=130)](https://github.com/infiniteluke) | [![Peter Sieg](https://avatars.githubusercontent.com/u/3128659?s=130)](https://github.com/chasingmaxwell)
```

Comment:
That empty first member is due to emdaer trying to pull from the `author` field as well

tbh, it's not going to be fixed until I have time to rewrite emdaer and I don't have a clue when that will happen, but I assume it won't be soon

Work arounds are to fill out the author field or to not use emdaer